content/about/about_people.rst

```diff
@@ -156,7 +156,7 @@ vikurtz
 .. raw:: html
 
   <br/>
-
+  
```

Comment:
Please remove this white space.

Reply:
The white space follows the formatting of all the other sections. Would you still like me to remove it? Also the issue was that I forgot .. class: no-breaks for Zach's github and irc

Reply:
It looks like it's a couple of spaces. I'm not talking about the newline. Does that make sense?